### DISCUSSION TITLE
精易论坛 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#33] [模板讨论]
精易论坛 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by LeticiaAnna** [comment]
签到结果：【签到成功】
交易币：【】，好评：【】，精币：【】，荣誉：【】，积分：【】

**Reply by wjf0214** [reply]
我看了下我的正常，重新抓下cookie吧。建议右键选中复制cookie，不要直接右键复制值试试。有空我再看看。。

**Comment by funnylook** [comment]
Failed at 1/4 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 301: Moved Permanently,
Request URL: https://bbs.125.la/plugin.php?id=dsu_paulsign%3Asign
